Traceback (most recent call last):
  File "/var/cr05_data/sabri/jonsf/PretrainingProject/PretrainingObjectives/pretraining/FuseAI/FuseLLM/./src/utils/forward_for_logits.py", line 187, in <module>
    logger = get_logger(__name__)
NameError: name 'get_logger' is not defined
